### ISSUE TITLE
MSHTML: Support for CSS content property

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4150]
**Reported by khsruru on 2014-05-22 12:29**
Hello, I’m Hyongsop Kim in Korea.
I have a question about reading css content properity.
By using content properity in css, developer can add a text in css.
But Nvda doesn’t read it in internet explorer.
Because content is a text, I think Nvda should read this text in internet explorer, too.
Please check this issue.
Thank you.

**Comment by bhavyashah** [comment]
@jcsteh @khsbory Could you please clarify the versions of Internet Explorer the reported issues applies to? In case this has been fixed by any chance in Internet Explorer 10/11, then we may safely close this ticket. Otherwise, we may need to mark this as P4.

**Comment by jcsteh** [comment]
STR:

1. Paste [the test case above](https://github.com/nvaccess/nvda/issues/4150#issuecomment-155320678) into an empty HTML file.
2. Open it in Internet Explorer.
3. Read the document in browse mode.
    - Expected: "before div"
    - Actual: "div"

Still persists in Internet Explorer Version: 11.0.16251.0, Update Versions: 11.0.41(KB4014661)

**Comment by Adriani90** [comment]
Still reproducible in NVDA 2019.1.1. I wonder if someone can test in the IE mode of Edge chromium if this issue occurs there as well?

**Comment by leonardder** [comment]
If I"m correct, this requires a change both in the MSHTML virtual buffer and on the MSHTML object level.

While I know that IE is still used in some corporate environments, IE is considered legacy by Microsoft. Given this is an issue that requires a lot of effort to implement, I'm tempted to close this as a wont fix. We can always reopen and remove the label if someone provides a pr.

**Comment by lukaszgo1** [comment]
@leonardder I have no interest in this particular issue, however my impression is that when issue is closed the PR for it wouldn't be accepted. Leaving it open as a P4 seems more reasonable.

**Comment by leonardder** [comment]
While a pr would be accepted, I think I agree on second thought, yes.

**Comment by Adriani90** [comment]
Support for IE ended completely in 2022 and is being automatically removed from machines that still have it after an Edge update in February 2023. Even though MSHTML will still be active until 2029 especially for backwards compatibility in IE mode in Edge, I don't expect any accessibility updates to IE or to IE mode in Edge. Closing as abandoned.
